web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/dialog/DialogController.kt

```diff
@@ -297,6 +303,7 @@ private fun lagFortsettDialog(request: FortsettDialogRequest, requestContext: Re
             .withKontorsperretEnhet(eldsteMelding.kontorsperretEnhet)
             .withTemagruppe(eldsteMelding.temagruppe)
             .withBrukersEnhet(eldsteMelding.brukersEnhet)
+            .withOppgaveId(eldsteMelding.oppgaveId)
```

Comment:
`Melding` som vi lager her propagerer ned til `HenvendelseUtils.createXMLHenvendelseMedMeldingTilBruker` ved kall til  `henvendelseUtsendingService.ferdigstillHenvendelse`.
 `oppgaveId` er ikke ibruk der, men for å sikre at vi ikke ved en tilfeldighet begynner å sende med oppgaveId så bør vi kanskje unngå å legge det til her.